test/Microsoft.Sbom.Api.Tests/Workflows/SbomConsolidationWorkflowTests.cs

```diff
@@ -148,7 +148,7 @@ public async Task RunAsync_MixOfValidAndInvalidInputSboms()
     {
         SetUpSbomsToValidate();
 
-        configurationMock.Setup(m => m.OutputPath).Returns(new ConfigurationSetting<string>("test-out-path"));
+        configurationMock.Setup(m => m.OutputPath).Returns(new ConfigurationSetting<string>());
```

Comment:
Is the setup needed if we no longer access it?

Reply:
It's still needed because we're setting outputPath to a temp dir for each validation run and then resetting it to null after